packages/create-cloudflare/src/helpers/command.ts

```diff
@@ -276,3 +300,28 @@ export const listAccounts = async () => {
 
 	return accounts;
 };
+
+/**
+ * Look up the latest release of workerd and use its date as the compatibility_date
+ * configuration value for wrangler.toml.
+ *
+ * If the look up fails then we fall back to a well known date.
+ *
+ * The date is extracted from the version number of the workerd package tagged as `latest`.
+ * The format of the version is `major.yyyymmdd.patch`.
+ *
+ * @returns The latest compatibility date for workerd in the form "YYYY-MM-DD"
+ */
+export async function getWorkerdCompatibilityDate() {
+	return runCommand("npm info workerd dist-tags.latest", {
```

Comment:
Should this run something different depending on the package manager that was used? Or is the assumption that npm will always be installed (which seems like a fair assumption)?

Reply:
Of course Yarn output is slightly different to npm and pnpm :sad:

Reply:
Added support for this and improved the date matching to accommodate yarn.
And added tests.